dev/ci/user-overlays/README.md

```diff
@@ -14,3 +14,5 @@ fi
 ```
 
 (`CI_PULL_REQUEST` and `CI_BRANCH` are set in [`ci-common.sh`](/dev/ci/ci-common.sh))
+
+Even if the shell scripts in this directory contain a shebang, they should not be directly executed.
```

Comment:
That's not what I meant by "update the README". I don't think this sentence is necessary: it is not dangerous to directly execute this shell scripts, it will just do nothing.
What I meant by "update the README" is to add the shebang in the quoted shell above: it says what `00669-maximedenes-ssr-merge.sh` contains but you've modified it.
BTW these overlay scripts are not meant to be kept (except this example one) so instead of patching them all, you could also simply remove them in a preliminary commit.

Reply:
Done.